config/manifests/bases/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -125,7 +125,7 @@ spec:
   minKubeVersion: 1.19.0
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.55.0
+  replaces: jaeger-operator.v1.57.0
```

Comment:
Shouldnt it be 56?

Reply:
Actually yes, not sure what happened here. Fixed!